.github/workflows/ci_common.yml

```diff
@@ -991,7 +991,7 @@ jobs:
           ref: ${{ inputs.prRef }}
 
       - name: Login to GitHub Container Registry
-        uses: docker/login-action@v1 
+        uses: docker/login-action@v2
```

Comment:
I am not sure why this is V1 for you, I have V2

Reply:
There's still one line with v1: https://github.com/devcontainers/ci/blob/main/.github/workflows/ci_common.yml#L994